packages/data-contract/src/actions/deployContract.js

```diff
@@ -0,0 +1,29 @@
+require('dotenv').config()
+const schema = require('../../schema.json')
+
+const { initClient, logInfo } = require('../utils')
+
+async function deployContract () {
+  logInfo('Client Initialization')
```

Comment:
Lets return basic console.log. It messes up with Dash SDK logging

Reply:
✅